test/performance/specs/post-editor.spec.js

```diff
@@ -270,7 +270,7 @@ test.describe( 'Post Editor Performance', () => {
 			const canvas = await perfUtils.getCanvas();
 
 			const paragraphs = canvas.getByRole( 'document', {
-				name: /Empty block/i,
+				name: /Block: Paragraph/i,
```

Comment:
I always forget the rule when we should make labels compatible with old branches, but this seems to be okay since tests create the fixtures.

Reply:
Yes, it shouldn't be needed since in both environments we create filled paragraphs now.